collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -33,15 +34,18 @@
 import org.odk.collect.android.preferences.PreferencesProvider;
 import org.odk.collect.android.storage.StoragePathProvider;
 import org.odk.collect.android.utilities.FileUtils;
+import org.odk.collect.android.utilities.LocaleHelper;
+import org.odk.collect.strings.LocalizedApplication;
 
 import java.io.ByteArrayInputStream;
 import java.io.File;
+import java.util.Locale;
 
 import javax.inject.Inject;
 
 import static org.odk.collect.android.preferences.MetaKeys.KEY_GOOGLE_BUG_154855417_FIXED;
 
-public class Collect extends Application {
+public class Collect extends Application implements LocalizedApplication {
```

Comment:
I looked at moving the `TranslationHandler` to the `strings` module but it would have meant moving all our code that deals with the custom locale. I ended up favouring this solution where we keep the locale stuff in `collect_app` and just have the `strings` code expect to be given a locale.